### PR TITLE
fix: assigning checks to groups and example [sc-0]

### PR DESCRIPTION
**Pull request by umutuzgur**
I hereby confirm that I followed the code guidelines found at [engineering guidelines](https://www.notion.so/checkly/Engineering-Guidelines-a3a165a203a04dc1a112f0e26b2f2d3f)

### Notes for the reviewer
* Adds fixes to check group patterns and follows the same matching style as project
* Adds an example to the example project
* How to add group id to checks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -512,6 +512,30 @@ new BrowserCheck('browser-check-1', {
 
 😔 sorry, no docs here yet. We are working on getting model right for Check groups.
```
Comment (clample) [inline README.md:513]: Should this be removed yet?

Comment (umutuzgur) [reply README.md:513]: I'm expecting some comments from Tim here given my writing skills are bare minimum :)

**Review by clample — APPROVED** [review]
It looks good to me. Maybe Tim will have input on the README and examples, though.

File: README.md (hunk 1 above)
Comment (tnolet) [reply README.md:513]: I will review and maybe push some changes to this branch

Comment (umutuzgur) [reply README.md:513]: Go for it

**Review by tnolet — APPROVED** [review]
LGTM

**Pull request merged**